README.md

```diff
@@ -1,46 +1,56 @@
-# 👷‍♀ ‍Compiled
+# Compiled
+
+**Build time atomic CSS in JS without the runtime cost.
+Bake,
+consume,
+extract.
+Everywhere.**
+
+[![Apache 2.0](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](./LICENSE)
+[![@compiled/core](https://img.shields.io/npm/v/@compiled/core.svg?style=flat-square)](https://www.npmjs.com/package/@compiled/core)
+[![PRs welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](./CONTRIBUTING.md)
 
 [Read the docs →](https://compiledcssinjs.com)
 
-## Get started
+## Usage
+
+```jsx
+import { styled } from '@compiled/core';
+
+// Define styles with the CSS prop
+<div css={{ color: 'purple' }} />;
+
+// Create a component with associated styles
+const StyledButton = styled.button`
+  color: ${(props) => props.color};
+`;
+
+// Define extra styles with the CSS prop
+<StyledButton css={{ fontSize: 16 }} color="pink" />;
+```
 
-**Install**
+## Installation
 
-> ⚠️ Work is in progress re-writing/architecting a new Babel plugin.
-> Use the nightly at your own risk until it is officially released.
+Install the core library:
 
 ```bash
-npm install @compiled/core@nightly @compiled/babel-plugin@nightly
+npm install @compiled/core --save
 ```
 
-**Configure [Babel](https://babeljs.io/docs/en/config-files)**
+Configure [Babel](https://babeljs.io/docs/en/config-files):
 
 ```json
 {
   "plugins": ["@compiled/babel-plugin"]
```

Comment:
Thoughts on using only the shorthand syntax:

```json
{	
  "plugins": ["@compiled"]
}
```

Reply:
Ahh thats a namespace actually. But loved your suggestion. Something smarter we can do 🤔

Reply:
its cute but we will be changing it to `@compiled/react/babel-plugin` so no point really.

Reply:
Yeah too cute I think. Agree